registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -83,6 +83,7 @@
     "sha256:763eac18e8b16f89a8e36f19b052d120231f6ef73f5bbab86457c4d83baa1ce3": ["v1.1.6"]
     "sha256:936dc17db44e0ade12ce57e6f0fff8d258bb847781befbd5d3cec6d21d838291": ["v1.10.0-beta.0"]
     "sha256:fe4a8024d4ebd86c630331bb8df8ff4e21cc5e1eeeee4112aa278c19b9a64135": ["v1.10.0-beta.1"]
+    "sha256:19493c79be104f86bd650f9137fe9cb61bf05081136eaaf8339415c131ba73dd": ["v1.10.0-rc.0"]
```

Comment:
`19493c79be10` matches the cluster-api-controller [staging repo for v1.10.0-rc.0](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/cluster-api-controller@sha256:19493c79be104f86bd650f9137fe9cb61bf05081136eaaf8339415c131ba73dd/details?invt=AbuOLA).